app/models/claims/allocation_filters.rb

```diff
@@ -47,7 +47,7 @@ def all_graduated_fees
 
     def all_risk_based_bills
       where(type: 'Claim::LitigatorClaim')
-        .where(offence_id: Offence.joins(:offence_class).where(offence_class: { class_letter: %w[E F H I] }))
+        .where(offence_id: Offence.joins(:offence_class).where(offence_classes: { class_letter: %w[E F H I] }))
```

Comment:
have you got a reference for this.

Reply:
http://api.rubyonrails.org/classes/ActiveRecord/QueryMethods.html#method-i-includes (at the end of this section)

> Note that includes works with association names while references needs the actual table name.

Reply:
https://stackoverflow.com/questions/16349195/how-to-join-an-associated-table-with-a-has-one-association